### PR TITLE
sequense_task_3_Gershkul_Oleksandr

### PR DESCRIPTION
**Pull request by DevASG**

### CODE REVIEW

File: sequence_task_3/student/include/impl.h [hunk 1]
```diff
@@ -11,4 +12,18 @@
...
+    std::istringstream iss(input);
+    std::deque<std::string> words;
+    std::copy(std::istream_iterator<std::string>(iss), std::istream_iterator<std::string>(),
+        std::front_inserter(words));
+
+    std::ostringstream oss;
+    std::copy(words.begin(), words.end(), std::ostream_iterator<std::string>(oss, " "));
+    std::string result { oss.str() };
+    if (!result.empty()) {
+        result.resize(result.size() - 1);
```
Comment (smay1613) [inline sequence_task_3/student/include/impl.h:26]: pop_back() is more appropriate method for this task

File: sequence_task_3/student/tests/unit_tests/src/maintest.cpp [hunk 2]
```diff
@@ -25,7 +25,21 @@ TEST(Reverse, TrailingSpaces)
...
 TEST(Reverse, Corner)
 {
-    const std::string input {""};
-    const std::string expected {""};
+    const std::string input { "" };
+    const std::string expected { "" };
+    EXPECT_EQ(reverseWords(input), expected);
+}
+
+TEST(Reverse, Corner_2)
```
Comment (smay1613) [inline sequence_task_3/student/tests/unit_tests/src/maintest.cpp:33]: Thanks, could you please create a new pull request for these tests?

**Review by smay1613 — COMMENTED** [review]
Nice!

File: sequence_task_3/student/tests/unit_tests/src/maintest.cpp (hunk 2 above)
Comment (DevASG) [reply sequence_task_3/student/tests/unit_tests/src/maintest.cpp:33]: Done
https://github.com/smay1613/LTC-STL-Student/pull/148

File: sequence_task_3/student/include/impl.h (hunk 1 above)
Comment (DevASG) [reply sequence_task_3/student/include/impl.h]: Done!

**Pull request closed without merging**